kubernetes/host.go

```diff
@@ -207,11 +207,11 @@ func HasMatchingVirtualServices(host Host, virtualServices []networking_v1alpha3
 
 // HasMatchingRegistryService returns true when the FDQN of the host param matches
 // with one registry status of the registryStatus param.
-func HasMatchingRegistryService(host string, registryServices []*RegistryService) bool {
+func HasMatchingRegistryService(itemNamespace string, host string, registryServices []*RegistryService) bool {
```

Comment:
Minor, I guess you brought the same name of the variable into the method signature.
I'd keep the variable as you added, but I'd try to normalize the signature to "namespace string".

It's just for coding style, to avoid to introduce too many edges in the naming.

Reply:
+1